native/cocos/bindings/manual/jsb_dragonbones_manual.cpp

```diff
@@ -434,6 +434,7 @@ bool register_all_dragonbones_manual(se::Object *obj) {
     __jsb_dragonBones_BaseFactory_proto->defineFunction("parseTextureAtlasData", _SE(js_cocos2dx_dragonbones_BaseFactory_parseTextureAtlasData));
 
     dragonBones::BaseObject::setObjectRecycleOrDestroyCallback([](dragonBones::BaseObject *obj, int type) {
+#if CC_PLATFORM != CC_PLATFORM_OPENHARMONY
```

Comment:
Why?

Reply:
It should be supported, but there should still be problems

Reply:
Please add todo for these codes.